frontend/src/components/VMediaInfo/VMetadata.vue

```diff
@@ -1,7 +1,15 @@
 <template>
   <dl v-if="isSm" class="metadata grid gap-8" :style="columnCount">
-    <div v-for="datum in metadata" :key="`${datum.label}`">
-      <dt class="label-regular mb-1 ps-1">{{ $t(datum.label) }}</dt>
+    <div v-for="datum in metadata" :key="datum.label">
+      <VSourceProviderTooltip
+        v-if="tooltipId(datum)"
+        :describedby="tooltipId(datum)"
+        class="label-regular mb-1 flex flex-row ps-1"
+        :datum="datum"
+      />
+      <dt v-else class="label-regular mb-1 flex flex-row ps-1">
+        {{ $t(datum.label) }}
+      </dt>
```

Comment:
The info icon is not horizontally centered. I tried the following in Chrome inspector, and it looked good:
* Added `items-center`
* Just in `xs`, I added a padding of 4px to top, right, and bottom in `dt` element. For bigger breakpoints, `ps-1` works correctly

Reply:
Sorry, @fcoveram , I didn't mean to re-request your review yet

Reply:
No problem!